target/linux/ipq807x/image/generic.mk

```diff
@@ -173,3 +173,20 @@ define Device/zyxel_nbg7815
 		kmod-bluetooth
 endef
 TARGET_DEVICES += zyxel_nbg7815
+
+define Device/yuncore_ax880
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Yuncore
+	DEVICE_MODEL := AX880
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@hk09
+	SOC := ipq8072
+	DEVICE_PACKAGES := ipq-wifi-yuncore_ax880
+	IMAGES := sysupgrade.bin factory.bin factory.ubi	
+	IMAGE/factory.bin := append-ubi | qsdk-ipq-factory-nand
+	IMAGE/factory.ubi := append-ubi
```

Comment:
No need to override the recipe defined in `Device/UbiFit` as its the same

Reply:
remove these lines?

```
	IMAGE/factory.bin := append-ubi | qsdk-ipq-factory-nand
	IMAGE/factory.ubi := append-ubi
```